### ISSUE TITLE
filter by process

### ISSUE THREAD

**Issue by jfromaniello** [MrRio/vtop#74]
It will be nice to filter by process either in the UI or like `vtop --pid $PID`

**Comment by adamyonk** [comment]
Yes! `f4` or `\` keys start a filtering search in htop, `/` starts a plain search.

**Comment by 0phoff** [comment]
I would love a search feature as well!
/ to search would be a nice addition, since you already use vim keys for other tasks

**Comment by autoteelar** [comment]
3 year later bump why is this not already a feature? i thought vtop was made to be more user friendly compared to the others...

**Comment by tuh8888** [comment]
See my comment in #161.
tldr: https://github.com/tuh8888/vtop/tree/search
